istioctl/pkg/validate/validate.go

```diff
@@ -59,6 +62,25 @@ Example resource specifications include:
 		constant.InstanceKind:          {},
 		constant.AttributeManifestKind: {},
 	}
+
+	istioSupportedServicePrefix = []string{
+		"http", "http-",
```

Comment:
Can we get these from pilot/pkg/model/service.go? Otherwise I think it will be easy for them to fall out of sync if we add/remove some

Reply:
Changed to use model.ParseProtocol